tools/minismackerplay.c

```diff
@@ -0,0 +1,290 @@
+#define SDL_MAIN_HANDLED
```

Comment:
In the long run, it would be nice to convert this tool to using smackw32 + brender.